lib/oli_web/live/delivery/table_model.ex

```diff
@@ -29,77 +26,85 @@ defmodule OliWeb.Delivery.SelectSource.TableModel do
         },
         %ColumnSpec{
           name: :requires_payment,
-          label: "Requires Payment",
-          render_fn: &__MODULE__.render_payment_column/3
+          label: "Payment",
+          render_fn: &__MODULE__.render_payment_column/3,
+          sort_fn: &__MODULE__.sort_payment_column/2
         },
         %ColumnSpec{
           name: :inserted_at,
           label: "Created",
-          render_fn: &SortableTableModel.render_inserted_at_column/3
+          render_fn: &SortableTableModel.render_inserted_at_column/3,
```

Comment:
Maybe not as part of this story, but we could tweak this logic a little bit to look more similar to the mocks in the ticket (i.e `"Created 2 days ago"` or `"Created March 10 by Mauricio"`), which I think is more user friendly.

Reply:
Sounds great! Will fix in the next commit, the missing part was initialize the session context in the live view (which has the user timezone, etc) to make it work properly with `FormatDateTime.date()`